WireAPI/Sources/WireAPI/APIs/AuthenticationAPI/AuthenticationAPI.swift

```diff
@@ -81,6 +81,6 @@ public protocol AuthenticationAPI: Sendable {
         emailCode: String,
         name: String,
         password: String
-    ) async throws
+    ) async throws -> ([HTTPCookie], UUID?)
```

Comment:
suggestion: please adjust the doc comment and maybe it makes sense to set labels in the tuple.

```suggestion
    /// - Returns: The abcd and efgh.

    func registerAccount(
        email: String,
        emailCode: String,
        name: String,
        password: String
    ) async throws -> (abcd: [HTTPCookie], efgh: UUID?)
```